From f1851ae40dd2e5a7000f24eaaab31ee8fc838577 Mon Sep 17 00:00:00 2001
From: Bernd Zeimetz <bernd@bzed.de>
Date: Sat, 23 Nov 2013 01:33:22 +0100
Subject: [PATCH] SConstruct: ignore .in files in debian subdirectories.

The debian packaging magic will handle all files in debian/*,
scons must not mess with them.
---
 SConstruct | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/SConstruct b/SConstruct
index c5120ae..290910b 100644
--- a/SConstruct
+++ b/SConstruct
@@ -1255,6 +1255,11 @@ def substituter(target, source, env):
 
 templated = glob.glob("*.in") + glob.glob("*/*.in") + glob.glob("*/*/*.in")
 
+# ignore files in subfolder called 'debian' - the Debian packaging
+# tools will handle them.
+templated = [ x for x in templated if not x.startswith('debian/') ]
+
+
 for fn in templated:
     builder = env.Command(source=fn, target=fn[:-3], action=substituter)
     env.AddPostAction(builder, 'chmod -w $TARGET')
-- 
1.8.4.rc3

